setup.py

```diff
@@ -1,10 +1,22 @@
-from setuptools import setup, find_packages
+import io
+from glob import glob
+from os.path import basename, dirname, join, splitext
+
+from setuptools import find_packages, setup
 
 PROJECT_URL = 'https://github.com/melexis/warnings-plugin'
 VERSION = '0.0.2'
 
+
+def read(*names, **kwargs):
+    return io.open(
+        join(dirname(__file__), *names),
+        encoding=kwargs.get('encoding', 'utf8')
+    ).read()
+
+
 setup(
-    name='warnings-plugin',
+    name='warnings',
```

Comment:
Ah no, there is nothing that prevents us from writing it like this. The name here does not have to match at all with the structure behind.

It just seems more logical to me as this makes the import statement exactly the same as the pypi package name. It also makes it consistent with the other mlx and swcc packages we have